azext_iot/deviceupdate/common.py

```diff
@@ -0,0 +1,43 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+shared: Define shared data types(enums) and constant strings.
+
+"""
+
+from enum import Enum
+
+
+class ADUPublicNetworkAccessType(Enum):
+    """
+    ADU public network access type.
+    """
+
+    ENABLED = "Enabled"
```

Comment:
I thought the standard was camelCase

Reply:
The recent specific occurrence where I requested you refactor to camelCase was for the json payload that gets returned to the user. In that regard we should use json typing best practices.

In this case, this is Python code, where Python typing best practice for enum is using all upper case names for enum members.